pkg/cmd/ssh_aws.go

```diff
@@ -64,15 +64,21 @@ func sshToAWSNode(nodeName, path, user, pathSSKeypair string, sshPublicKey []byt
 	a.createBastionHostSecurityGroup()
 	fmt.Println("")
 
+	// if global flag -c is specified call cleanupAwsBastionHost
```

Comment:
I would rather name the flag `cleanup` to make things more clear. Maybe we can also document it in the [readme](https://github.com/gardener/gardenctl/blob/master/README.md). We only have this for `aws`. In my opinion it would also make sense to add this for the other infrastructures, what do you think?

Reply:
sure.
 due to -c already use by no-cache, I change to -e as erase

Reply:
I agree with you, regarding the other infrastructures. I will work on that.